src/services/rollouts.js

```diff
@@ -284,11 +284,17 @@ module.exports = class RolloutsHelper {
 
 			//filter based on multiple resource_types
 			let resourceTypes =
-				queryParams.resource_type && queryParams.resource_type != '' ? queryParams.resource_type.split(',') : ''
-			if (resourceTypes) {
+				queryParams.resource_type && queryParams.resource_type !== ''
```

Comment:
@priyanka-TL please add comments in the code for explaining the code block

Reply:
done